examples/tornjak/keycloak/README.md

```diff
@@ -0,0 +1,114 @@
+# Deploy Tornjak with Authentication Enabled
+
+This example demonstrates Tornjak's capability to control access to the Frontend Application using
+User Management via [Keycloak](https://www.keycloak.org/).
+
+For more information regarding Tornjak User Management, please refer to the following documentation:
+
+* [Tornjak User Management](https://github.com/spiffe/tornjak/blob/main/docs/keycloak-configuration.md)
+* [Keycloak Configuration for Tornjak](https://github.com/spiffe/tornjak/blob/main/docs/keycloak-configuration.md)
+* [Detailed Blogs on Tornjak User Management](https://github.com/spiffe/tornjak/blob/main/docs/blogs.md)
+
+**NOTE:** This example works only with the Vanilla version of Kubernetes; it does not yet support Openshift.
+
+As part of the exercise, an instance of Keycloak is deployed to illustrate how to manage users' access to Tornjak.
+Once enabled, the Tornjak UI will redirect all authentication calls to the Keycloak instance to obtain the
+correct credentials. Authorization is based on these credentials and occurs at the Tornjak application level.
+
+## Deploy Keycloak Instance (Authentication Service)
+
+We will deploy the instance of Keycloak in the same namespace as the SPIRE Server
+
+```shell
+# Create a namespace to deploy Keycloak and SPIRE-server
+kubectl create namespace spire-server
+```
+
+```shell
+# Create a secret from the realm JSON file for Tornjak realm import
+kubectl create secret generic realm-secret -n spire-server --from-file=examples/tornjak/keycloak/tornjak-realm.json
+```
```

Comment:
Instead of creating the realm with a secret along with a large realm dump,
you can do it in the chart itself, all in one step, with something like:
https://github.com/spiffe/helm-charts-hardened/blob/main/examples/keycloak-config-cli-using-spire/keycloak-values.yaml#L70-L134

Only need to add the users/clients that are new to the realm rather then the entire realm export. Should be able to save almost 2000 lines that way too.

Reply:
Thank you for the suggestion @kfox1111, yes we can probably simplify further. I will take a look.

Reply:
Done!